### DISCUSSION TITLE
Access Request Reviewer RBAC Recommendations

### DISCUSSION THREAD

**Discussion by corkrean** [gravitational/teleport#9918] [Show and tell]
Teleport allows for two ways to assign access request approval permissions:

**Method One:** 
This is the Teleport recommended way to enable access request reviews

```
kind: role
metadata:
  name: example
spec:
  allow:
    review_requests:
      roles: ['example_role']
  rules:
    - resources:
      - access_request
      verbs:
      - list
      - create
      - read
      - delete
      #note that the "update" verb is not present
```
In this case, the command to approve an access request would be: `tsh request review --approve <request-id>
`

**Method Two (deprecated and not recommended):** 
Allow the approver to have `update` permissions on the access requests. e.g.:
```
- resources:
      - access_request
      verbs:
      - list
      - create
      - read
      - update
      - delete

```
In this case, the command to approve an access request would be: `tctl request approve <request-id>`

**Comment by dvillarraga** [comment]
Valid for Enterprise right? 

[Role Requests Documentation](https://goteleport.com/docs/access-controls/access-requests/role-requests/) says: "Roles containing a review_requests rule can only be used in Teleport Enterprise. In Teleport Community Edition, Access Requests must be approved by an admin running tctl on the Auth Server."

I am testing Community Edition and the only way I have found to approve a request is by running the approval in the Auth Server directly, e.g.:

`kubectl --namespace teleport exec deploy/teleport-auth -- tctl request approve 2c52607f-090c-4795-be7b-b34a52d87d1d`

Is there any way to promote a user as admin?
